src/looprestoration_tmpl_8.rs

```diff
@@ -1060,9 +1154,8 @@ unsafe extern "C" fn loop_restoration_dsp_init_arm(
             (*c).wiener[0] = Some(dav1d_wiener_filter7_8bpc_neon);
             (*c).wiener[1] = Some(dav1d_wiener_filter5_8bpc_neon);
         } else {
-            // TODO(perl): enable assembly routines here
```

Comment:
Should this change also be made for the 16-bit path?

Reply:
thanks, good catch! 😅